embedding-reports/host-the-report-engine-remotely/rest-api-reference/features-api/get-ai-response.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Get AI Response
+page_title: Get AI Response Request Explained
+description: "Learn how to use the Get AI Response endpoint of the Telerik Reporting REST Service and what response to expect."
+slug: telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/features-api/get-ai-response
+tags: get,ai,response
+published: True
+position: 2
+---
+
+# `Get AI Response` Overview
+
+The `Get AI Response` endpoint allows you to send a query to the AI assistant for a specific report document and receive a generated response. This enables users to interact with the report using natural language and obtain report insights powered by AI.
+
+## Request
+
+    POST /api/reports/clients/{clientId}/instances/{instanceId}/documents/{documentId}/ai/{threadId}/query
+
+__Path parameters__
+
+| Name | Type | Description |
+| ------ | ------ | ------ |
+|`clientId`|String|ID of the service client the AI thread is started in. Returned from [Register Client]({%slug telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/clients-api/register-client%}).|
```

Comment:
* Add the definite article and change "from" -> "by" in the rest of the descriptions too.
```suggestion
|`clientId`|String|The ID of the service client that started the AI thread. Returned by [Register Client]({%slug telerikreporting/using-reports-in-applications/host-the-report-engine-remotely/telerik-reporting-rest-services/rest-api-reference/clients-api/register-client%}).|
```